### PR TITLE
Update summary sheet when adding to an existing document

### PR DESCRIPTION
**Pull request by past**
This updates the summary sheet with the right references to the added sheet. It works in one of my test documents, but can't find another document at all. This seems unrelated to this change however, so let's get it reviewed.

### CODE REVIEW

File: spreadsheet.js [hunk 1]
```diff
@@ -32,6 +32,73 @@ async function createSpreadsheet(sheets, title, maxDate) {
...
 
+async function updateSummary(sheets, spreadsheetId, date) {
+    const title = getSheetTitle(date);
+    let result = await sheets.spreadsheets.get({
+        spreadsheetId,
+    });
+
+    let sheetId;
+    for (const { properties } of result.data.sheets) {
+        if (properties.title === "Summary") {
```
Comment (wisniewskit) [inline spreadsheet.js]: Time for a tiny style debate! I prefer an early-continue (or .filter) in a case like this to not indent so much code (not that I think it's worth changing here, but just a thought).

File: spreadsheet.js [hunk 2]
```diff
@@ -32,6 +32,73 @@ async function createSpreadsheet(sheets, title, maxDate) {
...
+            sheetId = properties.sheetId;
+            const valueInputOption = 'USER_ENTERED';
+            const requests = [];
+            const range = {
+                "sheetId": sheetId,
+                "startRowIndex": 1,
+                "endRowIndex": 2,
+                "startColumnIndex": 0,
+                "endColumnIndex": 12
```
Comment (wisniewskit) [inline spreadsheet.js]: Trailing comma, maybe?

File: spreadsheet.js [hunk 3]
```diff
@@ -431,4 +498,5 @@ module.exports = {
     createSpreadsheet,
     findOrCreateSheet,
     shareSheet,
+    updateSummary
```
Comment (wisniewskit) [inline spreadsheet.js]: Trailing comma maybe, part 3 (the comma-ing)?

File: spreadsheet.js (hunk 1 above)
Comment (past) [reply spreadsheet.js]: Yeah, that's how I had it in the beginning and it's how I wrote `findOrCreateSheet`, but having both a `continue` and a `break` seemed too much. I'll change it.

File: spreadsheet.js (hunk 2 above)
Comment (past) [reply spreadsheet.js]: We really need to set up eslint properly to enforce these things and not have to keep dealing with them.

Comment (miketaylr) [reply spreadsheet.js]: I filed https://github.com/past/tsci/issues/74 for discussion.

**Pull request merged**